src/test/java/org/cactoos/scalar/AndInThreadsTest.java

```diff
@@ -160,8 +161,8 @@ public void iteratesEmptyList() {
 
     @Test
     public void worksWithProc() throws Exception {
-        final List<Integer> list = new Synced<>(
-            new ArrayList<>(2)
+        final List<Integer> list = new Sticky<>(
```

Comment:
@victornoel change `new Sticky<>(new Synced<>(...))` by `new Solid<>(...)`.

Reply:
@fabriciofx I can't because it does not behave the same, `Solid` is `Synced` of `Sticky` while here I needed `Sticky` of `Synced`. Please raise an issue if you think they should behave the same.